modules/solid_mechanics/src/vectorpostprocessors/InteractionIntegralSM.C

```diff
@@ -263,27 +259,81 @@ InteractionIntegralSM::computeIntegral()
   fe->reinit(_current_elem);
 
   // calculate q for all nodes in this element
-  _q_curr_elem.clear();
   unsigned int ring_base = (_q_function_type == QMethod::Topology) ? 0 : 1;
 
-  for (unsigned int i = 0; i < _current_elem->n_nodes(); ++i)
+  for (unsigned int icfp = 0; icfp < _interaction_integral.size(); icfp++)
```

Comment:
I thought we were moving away form `unsigned int` loop indices. At the very least I'd use `std::size_t`